docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -196,6 +197,18 @@ make up
 NOTE: When Lisk Service is started again after a database reset, then the process to reindex all the data is initiated.
 This can be quite time-consuming when the chain is long, spanning over hours.
 
+== MySQL snapshot
+
+Configurations to sync Lisk Service from an existing snapshot can be used to speed up the sync process.
+It is possible to enable or create a snapshot of the MySQL database, and the available xref:{url_snapshot_config}[MySQL snapshot configurations] can be found in the Lisk Service configuration reference page.
+To change the configuration setting or to create a mySQL snapshot, set the `ENABLE_APPLY_SNAPSHOT` to `true`.
```

Comment:
Initially we added these 2 parameters as well `INDEX_SNAPSHOT_URL` and `ENABLE_SNAPSHOT_ALLOW_INSECURE_HTTP`. Is there any reason why we removed?

Reply:
These configurations have been moved to configuration/Index page, and here we are just explaining how to enable the configurations, and not the configuration itself. So the use of the definition is present here.